src/components/Select/Select.styles.js

```diff
@@ -1,14 +1,16 @@
 import styled from 'styled-components';
+import InputWrapper from '../../theme/inputs.styles';
 
-export const SelectWrapper = styled.div`
+export const SelectInputWrapper = styled(InputWrapper)`
   display: flex;
-  flex-flow: column;
-  position: relative;
+  flex-direction: column;
+
+  label {
+    font-weight: bold;
+  }
 
   select {
-    font-size: 1.6rem;
-    cursor: pointer;
-    color: var(--darkgrey);
+    appearance: none;
```

Comment:
Think we still need the font_size: 1.6rem so it's the same as TextInput and TextArea. It's better for accessibility. I don't think the placeholder styling in /theme/inputs.styles is working so will remove it.

Reply:
Yes you right, global styles do not apply to placeholder, even if its included as attritbute inside `<select>` tag.
I will have a look at it.

Regarding bold font , i removed that style.

Reply:
I founded some workaround for color of placeholder using `{ useState }`


const [color, setColor] = useState(`var(--darkgrey)`);

  function changeColor() {
    setColor(`var(--darkblue)`);
  }

then insite `<select>` tag i added:
`...
style={{ color }}
onChange={changeColor}
...`